contracts/src/bridge/SequencerInbox.sol

```diff
@@ -56,11 +58,85 @@ contract SequencerInbox is DelegateCallAware, GasRefundEnabled, ISequencerInbox
 
     mapping(bytes32 => DasKeySetInfo) public dasKeySetInfo;
 
+    /**
+     * @notice magic value used to skip validation against untrustedMessageCount
+     * @dev this is never a valid value for untrustedMessageCount since its bigger than uint64
+     */
+    uint256 public constant SKIP_UNTRUSTED_MSG_COUNT = uint256(type(uint64).max) + 1;
```

Comment:
I'm fine changing this value, but it's also safe to use `0` for it, because a batch will never truly have an end message count of zero, and a batch will only start with a message count of zero if its sequence number is zero, making it redundant in that case. There's a few different tests depending on this value I think (and the rollup itself), though we can change them of course.

Also, if you keep this as non-zero, we'll need to add it to a `postUpgradeInit` hook.